test/unwrapped_test.py

```diff
@@ -49,4 +49,9 @@ def test_unwrapped(name, env_module):
     env = wrappers.OrderEnforcingWrapper(env)
     env = wrappers.TerminateIllegalWrapper(env, 1.0)
 
+    if env.metadata["is_parallelizable"]:
+        env = conversions.aec_to_parallel(env)
+        env = conversions.parallel_to_aec(env)
+        env = conversions.turn_based_aec_to_parallel(env)
```

Comment:
Actually, neither parallel_to_aec nor turn_based_aec_to_parallel should check for "is_parallelizable", only aec_to_parallel should.